lib/Bio/DB/HTS.pm

```diff
@@ -1346,7 +1346,7 @@ sub hts_file { shift->{hts_file} }
 sub is_remote {
     my $self = shift;
     my $path = shift;
-    return $path =~ /^(http|https|ftp):/;
+    return $path =~ /^(http|https|ftp|s3):/;
```

Comment:
I think this is a peculiarity of the way the Test module evaluates stuff, and it just happens that "" is considered equal. I've updated the test to be `== 0` instead as I think that's clearer for the intended use. 

`ok(is_remote('/not'), undef)` fails, as does `ok(is_remote('/not'), ())`